llvm/test/CodeGen/WebAssembly/lrint-conv.ll

```diff
@@ -0,0 +1,62 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+
+; Tests for lrint and llrint, with both i32 and i64 checked.
+
+; RUN: sed 's/ITy/i32/g' %s | llc -mtriple=wasm32-unknown-unknown | FileCheck %s
+; RUN: sed 's/ITy/i64/g' %s | llc -mtriple=wasm32-unknown-unknown | FileCheck %s
```

Comment:
This isn't a great practice, I'd say tests editing their content is worse than just duplicating tests

Reply:
I guess it's fine for now, but I don't want to see this become widespread practice. In particular update_llc_test_checks will not handle this case